src/sage/interfaces/polymake.py

```diff
@@ -851,7 +850,7 @@ def new_object(self, name, *args, **kwds):
 
 
 ########################################
-## Elements
+#  Elements
 
```

Comment:
If you are willing, I would suggest
```
# --------
# Elements
# --------
```
as the standard style for header comments